BE/OwnershipAndControl/Executives.rdf

```diff
@@ -573,6 +623,24 @@
 		<skos:definition>indicates the situation that faciliates endorsement of the authorized party for some purpose</skos:definition>
 	</owl:ObjectProperty>
 	
+	<owl:ObjectProperty rdf:about="&fibo-be-oac-exec;isDirectorOf">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-pty-pty;actsOn"/>
+		<rdfs:label>is director of</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-be-oac-exec;BoardMember"/>
+		<rdfs:range rdf:resource="&fibo-be-oac-cpty;ControlledParty"/>
+		<owl:inverseOf rdf:resource="&fibo-be-oac-exec;hasDirector"/>
+		<skos:definition>indicates the organization that the board member oversees</skos:definition>
```

Comment:
So is "oversee" the same as "control"?

Reply:
Oversight is an aspect of control.  If the board fails to act responsibly when they become aware of certain situations within the organization they can be held liable for those failures.

Reply:
I agree they can be held liable, but that is not "control" as per the definition I quoted from &fibo-fnd-oac-ctl;Control"